packages/@pollyjs/core/tests/integration/configs.js

```diff
@@ -1,20 +1,27 @@
+const common = {
+  recordFailedRequests: true,
+  persisterOptions: {
+    host: ''
```

Comment:
We're using the testem proxy here which is mounted on /polly

Reply:
Was it mounted on the testem server to work around this?  I don't have the context, believe you mounted it there.

Reply:
I can do this is a separate PR.